apps/prairielearn/src/lib/db-types.ts

```diff
@@ -15,8 +15,8 @@ export type EnumGradingMethod = z.infer<typeof EnumGradingMethodSchema>;
 export const EnumJobStatusSchema = z.enum(['Running', 'Success', 'Error']);
 export type EnumJobStatus = z.infer<typeof EnumJobStatusSchema>;
 
-export const EnumModeSchema = z.enum(['Public', 'Exam', 'SEB']);
 export type EnumMode = z.infer<typeof EnumModeSchema>;
+export const EnumModeSchema = z.enum(['Public', 'Exam', 'SEB']);
```

Comment:
I don't think this is a good idea from a readability point of view, since the type depends on the const.